src/libs/fileDownload/index.desktop.ts

```diff
@@ -0,0 +1,35 @@
+import ELECTRON_EVENTS from '@desktop/ELECTRON_EVENTS';
+import type Options from '@desktop/electronDownloadManagerType';
+import CONST from '@src/CONST';
+import type {FileDownload} from './types';
+
+/**
+ * The function downloads an attachment on desktop platforms.
+ */
+const fileDownload: FileDownload = (url, fileName) => {
+    const options: Options = {
+        filename: fileName,
+        saveAs: true,
+    };
+    window.electron.send(ELECTRON_EVENTS.DOWNLOAD, {url, options});
+    return new Promise((resolve) => {
+        // This sets a timeout that will resolve the promise after 5 seconds to prevent indefinite hanging
+        const downloadTimeout = setTimeout(() => {
+            resolve();
```

Comment:
We should reject here and let the user know that something went wrong.

Reply:
@allroundexperts how should it look? Native fileDownload use RN alert to display errors/success, but it's [not implemented on the web/desktop](https://necolas.github.io/react-native-web/docs/react-native-compatibility/). I believe this could be considered a feature request.